benchmarks/graphics_pipeline/main.cpp

```diff
@@ -174,6 +188,7 @@ void ProjApp::Config(ppx::ApplicationSettings& settings)
 {
     settings.appName                    = "graphics_pipeline";
     settings.enableImGui                = true;
+    settings.allowThirdPartyAssets      = true;
```

Comment:
is this for future changes?

Reply:
I forgot to update this part, since I was using a third party asset but now it is not longer need it. Done.